src/components/ha-picture-upload.ts

```diff
@@ -60,13 +68,23 @@ export class HaPictureUpload extends LitElement {
           alt=${this.currentImageAltText ||
           this.hass.localize("ui.components.picture-upload.current_image_alt")}
         />
-        <ha-button
-          @click=${this._handleChangeClick}
-          .label=${this.hass.localize(
-            "ui.components.picture-upload.change_picture"
-          )}
-        >
-        </ha-button>
+        <div>
+          <ha-button
+            @click=${this._handleChangeClick}
+            .label=${this.hass.localize(
+              "ui.components.picture-upload.change_picture"
+            )}
+          >
+          </ha-button>
+          ${this.canDelete
```

Comment:
I don't think it is clear to a user what the delete button does, it looks like it does the same as the change picture button, but it deletes it from the server...

We should have a picker of current uplloaded files sometime 😓

Reply:
I would offer to remove the delete button, but it was just added on Matthias's request 😕 

The other weird thing about delete is that it's in a cancellable dialog. If you delete the picture, but then cancel the change, the picture becomes a broken link to a no longer existing image.

I assumed that more picture management functionality would maybe come someday in the future, but I felt maybe this was ok to go ahead without that, as that felt like might be a pretty big project with new APIs.

Reply:
@matthiasdebaat

Reply:
For the sake of not blocking this PR, I'm okay if the delete button is removed.

> We should have a picker of current uplloaded files sometime 😓

I agree

Reply:
> For the sake of not blocking this PR, I'm okay if the delete button is removed.

It's gone.